extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergTableAdapter.java

```diff
@@ -296,117 +274,37 @@ public TableDefinition definition(
             partitionSpec = table.spec();
         }
 
-        final IcebergInstructions userInstructions = instructions == null ? IcebergInstructions.DEFAULT : instructions;
-
         return fromSchema(schema,
                 partitionSpec,
                 userInstructions.tableDefinition().orElse(null),
                 getRenameColumnMap(table, schema, userInstructions));
     }
 
     /**
-     * Return {@link Table table} containing the {@link TableDefinition definition} of the Iceberg table.
-     *
-     * @return The table definition as a Deephaven table
-     */
-    public Table definitionTable() {
```

Comment:
I'm sad that the no-parameter version has disappeared for `definition()` and `table()`.  Returning the latest definition and table (as a static view) seems intuitive and keeps a low barrier to entry for new users.